suite-native/module-accounts-management/src/components/AccountDetailGraphHeader.tsx

```diff
@@ -63,17 +79,36 @@ const FiatBalance = () => {
     );
 };
 
-export const AccountDetailGraphHeader = ({ accountKey }: AccountBalanceProps) => {
+export const AccountDetailGraphHeader = ({ accountKey, tokenAddress }: AccountBalanceProps) => {
     const account = useSelector((state: AccountsRootState) =>
         selectAccountByKey(state, accountKey),
     );
+
+    const tokenInfo = useSelector((state: AccountsRootState) => {
```

Comment:
It would require new memoized selector but there is problem that memoization using proxy memoize is slow and expensive (my bad for choosing that lib, I want to write my own faster memo lib but not enough time 😅)

So this will actually call that expensive selector only when necessary.